java/component/keyple-core/src/main/java/org/eclipse/keyple/core/command/AbstractApduCommandBuilder.java

```diff
@@ -24,28 +24,41 @@
  *   <li>the built APDURequest,
  *   <li>the corresponding AbstractApduResponseParser class.
  * </ul>
+ *
+ * @since 0.9
  */
 public abstract class AbstractApduCommandBuilder {
 
   /**
-   * The reference field is used to find the type of command concerned when manipulating a list of
-   * abstract builder objects. Unfortunately, the diversity of these objects does not allow the use
-   * of simple generic methods.
+   * The reference field {@link SeCommand} is used to find the type of command concerned when
+   * manipulating a list of abstract builder objects. Unfortunately, the diversity of these objects
+   * does not allow the use of simple generic methods.
+   *
+   * @since 0.9
    */
   protected final SeCommand commandRef;
 
-  /** The command name (will appear in logs) */
+  /**
+   * The command name (will appear in logs)
+   *
+   * @since 0.9
+   */
   private String name;
 
-  /** the byte array APDU request. */
+  /**
+   * The byte array APDU request.
+   *
+   * @since 0.9
+   */
   protected ApduRequest request;
 
   /**
    * the generic abstract constructor to build an APDU request with a command reference and a byte
    * array.
    *
-   * @param commandRef command reference
+   * @param commandRef command reference (should not be null)
    * @param request request
```

Comment:
request is often set to null when building the commands, because the actual ApduRequest is defined later.
Don't know what to say...